kernel/src/arch/mod.rs

```diff
@@ -0,0 +1,27 @@
+pub mod x86_64;
+#[cfg(target_arch = "x86_64")]
+pub use self::x86_64::pci::pci::X86_64PciArch as PciArch;
+#[cfg(target_arch = "x86_64")]
+pub use self::x86_64::*; //公开x86_64架构下的函数，使外界接口统一
```

Comment:
It is not recommended to write code like this, because this will confuse readers and code-searching engine.
See: https://github.com/DragonOS-Community/DragonOS/pull/227#discussion_r1157455176